backend-project/small_eod/letters/models.py

```diff
@@ -12,23 +13,55 @@ class Direction(models.TextChoices):
         OUT = "OUT", "Sent"
 
     direction = models.TextField(
-        choices=Direction.choices, default=Direction.IN, max_length=3
+        choices=Direction.choices,
+        default=Direction.IN,
+        max_length=3,
+        verbose_name=_("Direction"),
+        help_text=_("Direction for letter."),
     )
 
-    date = models.DateTimeField()
+    date = models.DateTimeField(
+        verbose_name=_("Date"), help_text=_("Date of sending or receiving.")
+    )
     final = models.BooleanField()
 
-    name = models.CharField(max_length=256)
-    ordering = models.IntegerField(default=0)
-    comment = models.CharField(max_length=256)
-    excerpt = models.CharField(max_length=256)
-    identifier = models.CharField(max_length=256)
+    name = models.CharField(
+        max_length=256,
+        verbose_name=_("Description"),
+        help_text=_("Description of the letter."),
+    )
+    ordering = models.IntegerField(
+        default=0, verbose_name=_("Ordering"), help_text=_("Order of letter")
+    )
+    comment = models.CharField(
+        max_length=256, verbose_name=_("Comment"), help_text=_("Comment for letter.")
```

Comment:
Kiedy stosujesz kropkę na końcu?
Wcześniej na to zwróciłem uwagę i zamierzałem zbiorczo skorygować, bo nie zauważyłem od razu, ale teraz widzę, że piszesz to w różny sposób.

Reply:
Ogólnie stosowałam kropki w help_text. Jak nie ma, to przeoczyłam. Chcesz je zbiorczo usunąć, czy dodać tam gdzie brakuje? Bo jak coś to mogę sprawdzić wszystkie pliki, które robiłam i ujednolicić.

Reply:
Ja zwykle nie stosuje kropki, bo nie jest ona konieczna, skoro to jest tylko równoważnik i jeden zwrot. Ale szczerze mówiąc, to drobiazg i jestem w stanie zaakceptować dowolne spoje rozwiązanie.